bigquery_etl/shredder/config.py

```diff
@@ -369,9 +341,6 @@ def fields(self) -> Tuple[str, ...]:
         client_id_target(table="eng_workflow_stable.build_v1"),
         # other
         DeleteTarget(table="telemetry_stable.pioneer_study_v4", field=PIONEER_ID),
-        DeleteTarget(
-            table="telemetry_stable.pre_account_v4", field=DESKTOP_ECOSYSTEM_CLIENT_ID
-        ),
```

Comment:
I believe `pre_account` was added here in error. It does indeed have a field called `ecosystem_client_id`, but it is always set to "unknown". The `pre-account` ping was associated with a previous incarnation of "ecosystem telemetry" rather than AET.